app/src/main/res/xml/searchable.xml

```diff
@@ -4,9 +4,9 @@
 			android:includeInGlobalSearch="true"
 			android:label="@string/search_label"
 			android:searchSettingsDescription="@string/search_settings_description"
-			android:searchSuggestAuthority="com.boardgamegeek"
+			android:searchSuggestAuthority="com.boardgamegeek.provider"
```

Comment:
I don't think it's worth adding a different authority, but I'm interested to understand why this was changed

Reply:
I explained above, but as you said, not worth it.